haddock-api/src/Haddock/Interface/Create.hs

```diff
@@ -284,6 +285,8 @@ parseOption "hide"            = return (Just OptHide)
 parseOption "prune"           = return (Just OptPrune)
 parseOption "not-home"        = return (Just OptNotHome)
 parseOption "show-extensions" = return (Just OptShowExtensions)
+parseOption "print-runtime-reps" = return (Just OptPrintRuntimeRep)
+parseOption "print-explicit-runtime-reps" = return (Just OptPrintRuntimeRep)
```

Comment:
Is there a reason for having two flags with the same effect? Why not just standardize on a single flag?

Reply:
No particular reason. I want this flag to be called `print-runtime-reps`, but I anticipate some confusion with GHC's `-fprint-explicit-runtime-reps` (cf. Cabal's `hyperlink-source` option vs Haddock's `hyperlinked-source` option). Do you have an opinion one way or another?

Reply:
My inclination would be to have `-fprint-explicit-runtime-reps` be the shared name of the flag across GHC and Haddock. I suppose I would be fine with picking `-fprint-runtime-reps` to be the name of the Haddock flag as well, but I definitely think we shouldn't come up with two different ways of referring to the same flag—that just seems like a recipe for confusion.